bundle/Command/LegacyInitCommand.php

```diff
@@ -0,0 +1,138 @@
+   needs to completely replace ezpublish-legacy package for different reasons.
+
+<comment>NOTE: Once this is ran you'll get instructions on which commands to run to setup symlinks once you have populated the folders.</comment>
+EOT
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->createDirectories($input, $output);
+        $this->updateComposeJson($output);
+    }
+
+    protected function createDirectories(InputInterface $input, OutputInterface $output)
+    {
+        $srcArg = rtrim($input->getArgument('src'), '/');
+
+        /**
+         * @var \Symfony\Component\Filesystem\Filesystem
+         */
+        $filesystem = $this->getContainer()->get('filesystem');
+
+        $filesystem->mkdir([
+            $srcArg,
+            "$srcArg/design",
+            //"$srcArg/AppBundle",
+            "$srcArg/AppBundle/ezpublish_legacy",
+            "$srcArg/settings",
+            "$srcArg/settings/override",
+            "$srcArg/settings/siteaccess",
+        ]);
+
+        $output->writeln(<<<EOT
+
+The following folders should have been created (or where already present):
+- src/AppBundle/ezpublish_legacy  _(for extensions)_
+- src/legacy_files/design
+- src/legacy_files/settings/override
+- src/legacy_files/settings/siteaccess
+
+Move over files and directories from older installation and afterwards run the folling commands to setup symlinks into
+legacy install:
+- <info>ezpublish:legacy:symlink</info>
+- <info>ezpublish:legacybundles:install_extensions</info>
+- <info>ezpublish:legacy:assets_install</info>
+
+
+EOT
+        );
+    }
+
+    protected function updateComposeJson(OutputInterface $output)
+    {
+        $errOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
+        $updateComposerJson = false;
+        $composerJson = json_decode(file_get_contents('composer.json'), true);
+        if ($composerJson === null) {
+            $errOutput->writeln('Error: Unable to parse composer.json');
+            return;
+        }
+
+        if (!array_key_exists('legacy-scripts', $composerJson['scripts'])) {
+            $legacy_scripts = [
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installAssets',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installLegacyBundlesExtensions',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::generateAutoloads',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::symlinkLegacyFiles',
+            ];
+            $composerJson['scripts'] = array_merge(['legacy-scripts' => $legacy_scripts], $composerJson['scripts']);
+            $updateComposerJson = true;
+        }
+
+        if (!array_key_exists('symfony-scripts', $composerJson['scripts'])) {
+            $composerJson['scripts']['symfony-scripts'] = ['@legacy-scripts'];
```

Comment:
In this case we should probably either write warning saying symfony scripts should be added post-install and post-update, or we should try to do that here(?)